CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Remove copyright year and sign from license and attributions
 
 ### Fixed
+- Fix homepage image links that were too large in Internet Explorer [styleguide-only]
```

Comment:
@maya hrmmm...  all versions meaning IE[X] - ? only the one's we support? Looks like you mentioned testing only on IE9 - 10.